Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -1,7 +1,14 @@
 { Expression } = require './expression'
+{ FunctionRef } = require './reusable'
+{ decimalAdjust } = require '../util/math'
 { ValueSet, Code } = require '../datatypes/datatypes'
+{ Exception } = require '../datatypes/exception'
 { build } = require './builder'
+ucum = require  'ucum'
 
+module.exports.IncompatibleTypesException = IncompatibleTypesException = class IncompatibleTypesException extends Exception
+  constructor: (@a , @b , e) ->
+    super("Incompatible Types " + @a + " " + @b, e)
```

Comment:
This error appears to be thrown when a unit is not a UCUM unit. See screen shot.

![image](https://user-images.githubusercontent.com/11574199/28939010-e2f37392-785d-11e7-83b1-f87ca0ec5e6e.png)

This results in a confusing error message to the user: the types are compatible (they're both `wks`), they're just not correct.

Can there be a "bad ucum unit" exception that gets thrown instead in these situations?